src/background/background.js

```diff
@@ -50,6 +52,42 @@ browser.runtime.onMessage.addListener((request, sender, sendResponse) => {
   }
 })
 
+/**
+ * changes the favicon if above a certain threshold
+ * 
+ * Defined, used in background.js
+ */
+async function changeFavicon () {
+  const currentWindow = await browser.tabs.query({ active: true, currentWindow: true })
+  const currentWindowId = currentWindow[0].id
+  const currentUrl = currentWindow[0].url
+  const currentHostName = getHostname(currentUrl)
+  /**
+   * swaps the favicon
+   */
+  const swapFavicon = async () => {
+    var evidence = await evidenceIDB.get(currentHostName) 
+    if (evidence == undefined) evidence = {}
+    var numEvidence = 0
+    for (let typ of Object.values(evidence)) {
+      for (let lst of Object.values(typ)) {
+        numEvidence += Object.keys(lst).length
+      }
+    }
+    if (numEvidence > EVIDENCE_THRESHOLD) {
+      // change the path when we get the right favicon to switch to
+      browser.browserAction.setIcon({tabId: currentWindowId, path:"../assets/favicon2.svg"})
+    } else {
+      // Change it back to the original. Sometimes quickly changing a webpage after load
+      // keeps the now-incorrect favicon
+      browser.browserAction.setIcon({tabId: currentWindowId, path:"../assets/favicon.svg"})
+    }
+  }
+  // timeout set to 5 seconds to allow for initial third parties loaded.
+  setTimeout(swapFavicon, FIVE_SEC_IN_MILLIS)
+}
+browser.webNavigation.onDOMContentLoaded.addListener(changeFavicon)
```

Comment:
Add a comment highlighting that this is opening a listener and calling the above function.